cmd/buildah/addcopy.go

```diff
@@ -52,13 +55,15 @@ func init() {
 	addFlags.SetInterspersed(false)
 	addFlags.BoolVar(&addOpts.addHistory, "add-history", false, "add an entry for this operation to the image's history.  Use BUILDAH_HISTORY environment variable to override. (default false)")
 	addFlags.StringVar(&addOpts.chown, "chown", "", "set the user and group ownership of the destination content")
+	addFlags.StringVar(&addOpts.ignore, "ignore", "", "path to .dockerignore file")
 	addFlags.BoolVarP(&addOpts.quiet, "quiet", "q", false, "don't output a digest of the newly-added/copied content")
 
 	// TODO We could avoid some duplication here if need-be; given it is small, leaving as is
 	copyFlags := copyCommand.Flags()
 	copyFlags.SetInterspersed(false)
 	copyFlags.BoolVar(&copyOpts.addHistory, "add-history", false, "add an entry for this operation to the image's history.  Use BUILDAH_HISTORY environment variable to override. (default false)")
 	copyFlags.StringVar(&copyOpts.chown, "chown", "", "set the user and group ownership of the destination content")
+	copyFlags.StringVar(&copyOpts.ignore, "ignore", "", "path to .dockerignore file")
```

Comment:
Please rename this to `dockerignore`, so that if we ever add an option to specify values like the ones we'd see in the file, but specified directly on the command line, we can call that `ignore`.

Reply:
Or `ignorefile`, I guess.

Reply:
I prefer `--ignorefile`